airbyte-integrations/connectors/source-amazon-ads/source_amazon_ads/spec.py

```diff
@@ -61,7 +63,17 @@ class Config:
         description="Start date for collectiong reports, should not be more than 60 days in past. In YYYY-MM-DD format",
         examples=["2022-10-10", "2022-10-22"],
     )
-    host: str = Field(None, alias="_host")
+
+    region: AmazonAdsRegion = Field(
```

Comment:
Please make AmazonAdsRegion.NA as default value to make this config compatible with previous version